lib/satismeter/test.js

```diff
@@ -106,6 +106,36 @@ describe('SatisMeter', function(){
           }
         });
       });
+
+      it('should send translation', function(){
```

Comment:
'should send custom traits', since we're not doing anything specific for language, or email (the one below). I would also combine this and the test below into one case.